src/sidebar/components/GroupList/GroupListItem.tsx

```diff
@@ -15,6 +18,27 @@ import { useSidebarStore } from '../../store';
 import { copyPlainText } from '../../util/copy-to-clipboard';
 import MenuItem from '../MenuItem';
 
+function GroupIcon({ type }: { type: GroupType }) {
+  const title =
+    type === 'open'
+      ? 'Public group'
+      : type === 'restricted'
+        ? 'Restricted group'
+        : 'Private group';
+
+  return (
+    <div
+      className="text-color-text-light"
+      title={title}
+      data-testid="group-icon"
+    >
+      {type === 'open' && <GlobeAltIcon />}
+      {type === 'restricted' && <GlobeAltLockIcon />}
+      {type === 'private' && <LockIcon />}
+    </div>
```

Comment:
Do we actually need this container element or can the props just be set on the icons themselves. Icons do forward extra props to the inline `<svg>` element, but there may be another reason I'm missing.

Reply:
I added the container because for some reason the title doesn't work otherwise. I'm not sure if it's expected in SVGs, or it's a bug somewhere.

I can see the attribute being set in the actual DOM node, but the tooltip is not displayed.

Let me try with other browsers and I'll google a bit to make sure.

Reply:
Yeah, I'm reading `SVG`s do not use the `title` attribute, but a `<title />` element that makes the browser render a tooltip.

So I'll merge it as is.